uploadChromiumBuildToCloudDevelopment.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+echo Uploading chromium build from local dir to cloud-dev
+echo
+
+# Always use ReplayDevelopment aws profile, no matter what the
+# default is.
+export AWS_PROFILE=ReplayDevelopment
+echo AWSProfile: ${AWS_PROFILE}
+
+# Use the current dir as reference point.
+DIR=$(pwd)
+TARGET_DIR="${DIR}/out/Release"
+S3_DIR_PATH="s3://recordreplay-us-east-2-dev/builds"
+
+echo CurrentDir: ${DIR}
+
+# Read the build id
+BUILD_ID_FILE=base/record_replay_driver.cc
+BUILD_ID=$(grep gBuildId "${BUILD_ID_FILE}" | sed 's/^.*"\([^"]*\)".*$/\1/')
+
+S3_BUILD_PREFIX="${S3_DIR_PATH}/${BUILD_ID}"
+
+echo BuildId: ${BUILD_ID}
+echo S3TargetPath: ${S3_BUILD_PREFIX}
+
+# Make a temp-dir to store the distribution
+TMP_DIR=$(mktemp -d)
+echo TmpDir: ${TMP_DIR}
+
+echo
+echo RUNNING copyLinuxBuild.js
+mkdir "${TMP_DIR}/replay-chromium"
+node copyLinuxBuild.js "${TARGET_DIR}" "${TMP_DIR}/replay-chromium"
+
+echo
+echo ARCHIVING "${TMP_DIR}/replay-chromium"
+(cd "${TMP_DIR}" && tar cfJ ./archive.tar.xz replay-chromium)
+
+echo COPYING to "${S3_BUILD_PREFIX}.xz"
+aws s3 cp "${TMP_DIR}/archive.tar.xz" "${S3_BUILD_PREFIX}.xz"
+
+# No, I'm not going to rm -rf the tmpdir.  That command should never appear
+# in shell scripts where the path is a raw variable with no literal path
```

Comment:
Since you're calling `mktemp -d` can't we assume that the file is in `/tmp`? Couldn't we construct an absolute path so that we can safely delete it?

Reply:
I'd have to extract the dirname, split it away from the '/tmp', then call `rm -rf /tmp/${SUBDIR}`, which is all doable, but it doesn't matter too much for now.  If enough people use this often enough where some tmpdir tax starts being significant, we can revisit.

I've gotten burned by this once, VERY BADLY, so I just avoid doing it :)